src/Components/Sidebar.lua

```diff
@@ -26,9 +26,17 @@ local function Sidebar(props: Props, hooks: any)
 		if node.instance and node.instance:IsA("ModuleScript") and node.name:match(constants.STORY_NAME_PATTERN) then
 			props.selectStorybook(node.storybook)
 			props.selectStory(node.instance)
-			setActiveNode(node)
+			setActiveNode(function(prevNode)
+				if prevNode then
+					if prevNode.instance == node.instance then
+						return nil
+					end
+				end
+
+				return node
+			end)
```

Comment:
Maybe something like this too?
```suggestion
			setActiveNode(function(prevNode: Explorer.Node)
				return if prevNode.instance ~= node.instance then node else nil 
			end)
```

Reply:
Also is instance comparison necessary or could we get away with `prevNode ~= node`?